airesources/PHP/MyBot.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+require_once __DIR__.'/src/bootstrap.php';
+
+$connection = new Connection();
+$game = new Game('Terminator', $connection);
+$turn = 0;
+
+//60s time for game initialization;
```

Comment:
I think I'm missing something. Is map actually parsed at this point? I only see it being parsed in updateMap() function but it hasn't been called here yet. I.e. how do you use the initial map here?

Reply:
@Snaar I thought as soon as map is parsed, the 60s are not allowed anymore. So when exactly is it allowed to wait 60s?

Reply:
It's actually as soon as you send your bot name over is when the 60s timer stops. What happens is you get pre-game map first, along with map sizes and your player id. Here is where you can take 60s. Then you send your bot name over and then the game will send you the first turn map - which actually would look just like the pre-game map.

Check out how some other kits are doing it. #151 has the list of kits that are doing it right currently.

Reply:
(This is the only remaining issue before I'd merge it in btw. Rest looks good.)